lib/secret_code.rb

```diff
@@ -0,0 +1,15 @@
+class SecretCode
+  attr_reader :code
+
+  def initialize
+    @code = gen_secret_code
+  end
+
+  def gen_secret_code
+    randomness = Random.new
+    return (1..4).map { |_| randomness.rand(1..6) }
+    puts(blue_b('The secret code has been generated, you may begin . . .'))
```

Comment:
`gen_secret_code` is violating SRP. Is there another way you could handle outputting to the console without adding that behavior to this method?